.github/workflows/payment-test-suite.yml

```
@@ -0,0 +1,523 @@
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: body
+            });
+
+  deploy-staging:
+    name: Deploy to Staging
+    runs-on: ubuntu-latest
+    needs: [test-summary]
+    if: |
+      github.ref == 'refs/heads/develop' &&
```

actionlint (1.7.7)

property "lint" is not defined in object type {test-summary: {outputs: {}; result: string}}

(expression)